libs/design/src/scss/theming/_variables.scss

```diff
@@ -87,9 +87,22 @@ $daff-bronze: (
 	100: #1a0f00,
 );
 
+$daff-green: (
+	10: #dafcde,
+	20: #a9fbb5,
+	30: #4ee56e,
+	40: #35c457,
+	50: #00ab3d,
+	60: #00872f,
+	70: #0e6726,
+	80: #0f4c1e,
+	90: #0c3515,
+	100: #07230d,
+);
+
 $daff-gray: (
 	0: #ffffff,
-	10: #f7f7f7,
+	10: #fafafa,
```

Comment:
Different PR please with the motivation for the change.

Reply:
removed